mne/tests/test_annotations.py

```diff
@@ -128,6 +129,16 @@ def test_annotations():
     assert raw_read.annotations is None
 
 
+@testing.requires_testing_data
+def test_read_brainstorm_annotations():
+    """Test reading for Brainstorm events file"""
+    fname = op.join(data_dir, 'events_sample_audvis_raw_bst.mat')
+    annot = read_brainstorm_annotations(fname)
+    assert len(annot) == 238
+    assert annot.onset.min() > 40  # takes into account first_samp
+    assert np.unique(annot.description).size == 5
```

Comment:
@annesodub can you also test with orig_time parameter in read_brainstorm_annotations

something like:

```
raw = ...
orig_time = raw.info['meas_date']
annot = read_brainstorm_annotations(fname, orig_time)
assert annot.orig_time == orig_time 
```

Reply:
Is there no other way to create a valid `orig_time`? rather than loading it from a raw file unrelated to this test?

Reply:
really good point. Please set it the the tuple value you get from the sample file. it's faster.

Reply:
I don't really know how to create a valid meas_date without loading data.. 
Alex, what do you mean by "Please set it the the tuple value you get from the sample file. it's faster."

Reply:
please add these lines:

```
    # Now test with orig_time
    orig_time = np.array([1038942070, 720100], dtype=np.int32)
    annot = read_brainstorm_annotations(fname, orig_time=orig_time)
    orig_time_as_scalar = orig_time[0] + orig_time[1] / 1000000.
    assert_array_equal(annot.orig_time, orig_time_as_scalar)
```

Reply:
then you'll need to update whats_new.rst file and it's good to go from my end.